src/main/java/edu/yale/its/tp/cas/client/filter/CASFilter.java

```diff
@@ -22,6 +22,7 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 
+import edu.yale.its.tp.cas.util.Configuration;

```

Comment:
Nope, you are correct and understood my question. I mistakenly assumed CASReceipt was concocted from the http request.